### DISCUSSION TITLE
Minify resources in wwwroot?

### DISCUSSION THREAD

**Discussion by Kixunil** [btcpayserver/btcpayserver#2320] [Ideas / Feature Requests]
I noticed that some resources in `wwwroot` are not minified. Might be nice to do so (during build) - could have impact on loading speed over Tor.

I can obviously do it myself but having it in the official code would benefit more people.

**Comment by Kukks** [comment]
We have a bundler that creates minified bundles from the js/css files in
wwwroot. There may be a few locations where we did not use the bundling
mechanism (in cases where there is only 1/2 js files used)

**Reply by Kixunil** [reply]
Yeah, I noticed some are minified already. Looked a bit like a bug but wasn't sure. Also this is currently not something crazy important for me. Just nice to have.

**Reply by woutersamaey** [reply]
Does the bundler use PurgeCSS? If not, this has huge potential as it removes all unused CSS by scanning the template files. Bootstrap can be quite bloated
